src/platform-includes/sourcemaps/upload/cli/javascript.mdx

```diff
@@ -47,7 +47,7 @@ Visit the [auth token user settings page](https://sentry.io/settings/account/api
 
 <Note>
 
-`sentry-cli >= 2.17.0` is required to upload artifact bundles.
+Uploading artifact bundles is supported from **Sentry CLI** version `2.17.0` onwards and the **Sentry JavaScript SDK** needs to be on version `7.47.0` or later.
```

Comment:
---
Note

Uploading artifact bundles requires:
- `sentry-cli` version >= `2.17.0`
- `${sdkName}` version >= `7.47.0`

/Note

${sdk-name} is just a placeholder because I think we can dynamically render the specific sdk name the user has selected